back-end/app.js

```diff
@@ -35,6 +36,32 @@ app.post("/room", function (req, res) {
   res.send({ roomId });
 });
 
+app.post("/new-user", function (req, res) {
+  req.session.groupID = "03307"; // this is temporary until Issue #73
+  const name = req.body.userName;
+  const newUser = new User({groupId:req.session.groupID, name: name, dishPreferences: []});
+  newUser.save((err, result) => {
+    if (err) {
+      console.log(err);
+      res.status(404);
```

Comment:
404 means page not found- perhaps we could use a 500 internal server error instead?
```suggestion
      res.status(500);
```